Upload_files_to_s3/README.md

```diff
@@ -0,0 +1,3 @@
+# Upload files & folders from your machine to Amazon S3
```

Comment:
Please add `usage` , `requirements` section.
See other scripts for reference.

Reply:
added